lib/tlog/es_json_reader.c

```diff
@@ -230,6 +231,45 @@ tlog_es_json_reader_init(struct tlog_json_reader *reader, va_list ap)
         goto error;
     }
 
+    /* Allow kerberos (negotiate) authentication */
+    rc = curl_easy_setopt(es_json_reader->curl, CURLOPT_HTTPAUTH, CURLAUTH_ANY);
+    if (rc != CURLE_OK && rc != CURLE_UNKNOWN_OPTION && rc != CURLE_NOT_BUILT_IN) {
+        grc = TLOG_GRC_FROM(curl, rc);
+        goto error;
```

Comment:
Minor style change request: Could you do the following instead?
~~~
   rc = curl_easy_setopt(es_json_reader->curl, CURLOPT_HTTPAUTH, CURLAUTH_ANY);
    if (rc != CURLE_OK)
        if (rc != CURLE_UNKNOWN_OPTION && rc != CURLE_NOT_BUILT_IN) {
            grc = TLOG_GRC_FROM(curl, rc);
            goto error;
   }
~~~